client/styles/components/_sketch-list.scss

```diff
@@ -33,4 +33,29 @@
   @include themify() {
     @extend %icon;
   }
-}
\ No newline at end of file
+}
+
+.visibility-toggle .sketch-list__trash-button {
+  @include themify() {
+    margin: 0;
+    padding: 0;
+    visibility: hidden;
+    width: 20px;
+  }
+}
+
+.visibility-toggle:hover .sketch-list__trash-button {
+   @include themify() {
+    background-color: transparent;
+    border: none;
+    cursor: pointer;
+    margin: 0;
+    padding: 0;
+    visibility: visible;
```

Comment:
This will also need to be changed because of my comment about accessible hidden elements.